examples/twr_aloha/control.c

```diff
@@ -0,0 +1,268 @@
+/*
+ * Copyright (C) 2020 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     examples
+ * @{
+ *
+ * @file
+ *
+ * @author      Francisco Molina <francois-xavier.molina@inria.fr>
+ *
+ * @}
+ */
+
+#include <assert.h>
+#include <math.h>
+#include <string.h>
+#include <stdio.h>
+
+#include "uwb/uwb.h"
+#include "uwb_rng/uwb_rng.h"
+#include "dpl/dpl.h"
+#include "control.h"
+
+#include "shell_commands.h"
+#include "shell.h"
+#include "xtimer.h"
+
+#ifndef LOG_LEVEL
+#define LOG_LEVEL LOG_DEBUG
+#endif
+#include "log.h"
```

Comment:
In fact, this is an example application, so I would completely replaces the calls to `LOG_DEBUG` functions with plains `puts`/`printf`. And also remove the `LOG_*` defines/include used here.

```suggestion
```

Reply:
ok